### PR TITLE
Admin Site Settings Links changes

### PR DESCRIPTION
**Pull request by alihadi-mazeh**
image

### CODE REVIEW

File: app/javascript/hooks/forms/admin/site_settings/useLinksForm.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { useCallback, useMemo } from 'react';
 
 export function useLinksFormValidation() {
   return useMemo(() => (yup.object({
-    value: yup.string().url(),
+    value: yup.string().url('Invalid URL'),
```
Comment (farhatahmad) [inline app/javascript/hooks/forms/admin/site_settings/useLinksForm.js:9]: translation?

File: app/javascript/components/admin/site_settings/administration/LinksForm.jsx [hunk 2]
```diff
@@ -17,17 +15,16 @@ export default function LinksForm({ id, value, mutation: useUpdateSiteSettingsAP
...
-          { t('admin.site_settings.administration.change_url') }
-        </Button>
-      </InputGroup>
+      <FormControl
+        field={fields.value}
+        aria-describedby={`${id}-submit-btn`}
+        type="text"
+        noLabel
+      />
+      <Button id={`${id}-submit-btn`} variant="brand" type="submit" disabled={updateSiteSettingsAPI.isLoading} className="mb-2 float-end">
```
Comment (hadicheaito1) [inline app/javascript/components/admin/site_settings/administration/LinksForm.jsx:24]: Not important, but could you move `className` to be right after `id`. It's just aesthetically nicer to read/good practice

**Comment by scouillard** [comment]
you can do `git rebase -i upstream/master` to merge master into this branch

File: app/assets/locales/en.json [hunk 3]
```diff
@@ -25,6 +25,7 @@
   "created_at": "Created at",
   "no_result_search_input": "Could not find any results for \"{{ searchInput }}\"",
   "action_permanent": "This action cannot be undone.",
+  "invalid_url": "Invalid URL",
```
Comment (farhatahmad) [inline app/assets/locales/en.json:28]: Can we try placing this somewhere a bit more inline with how the others are placed?

File: app/assets/locales/en.json [hunk 4]
```diff
@@ -20,6 +20,7 @@
   "or": "Or",
   "online": "Online",
   "help_center": "Help Center",
+  "invalid_url": "Invalid URL",
```
Comment (farhatahmad) [inline app/assets/locales/en.json:23]: Still not in the right place

File: app/javascript/hooks/forms/admin/site_settings/useLinksForm.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ import { useCallback, useMemo } from 'react';
 
 export function useLinksFormValidation() {
   return useMemo(() => (yup.object({
-    value: yup.string().url('invalid_url'),
+    value: yup.string().url('forms.validations.url.invalid'),
```
Comment (hadicheaito1) [inline app/javascript/hooks/forms/admin/site_settings/useLinksForm.js:9]: Your missing the translation here, what you have will just print `forms.validations.url.invalid`  as a literal string

**Review by farhatahmad — APPROVED** [review]

**Pull request merged**